EndlessClient/Dialogs/ListDialogItem.cs

```diff
@@ -69,6 +66,13 @@ public int OffsetY
             }
         }
 
+        public int HighlightWidthOverride
```

Comment:
1. Can this be an auto property? Trying to avoid the property-with-backing-field paradigm unless the getter/setter have logic associated with the property and it is unavoidable.
2. Can you make this an `Optional<int>` or `int?` instead of using the magic value `-1` to indicate that it is unset?